Assets/Python/Screens/PlatyBuilder/WBPlotScreen.py

```diff
@@ -619,13 +623,13 @@ def handleInput(self, inputClass):
 		elif inputClass.getFunctionName().find("BaseYield") > -1:
 			i = YieldTypes(inputClass.getData2())
 			if inputClass.getData1() == 1030:
-				CyGame().setPlotExtraYield(pPlot.getX(), pPlot.getY(), i, iChange)
+				CyGame().setPlotExtraYield(pPlot.getX(), pPlot.getY(), i, abs(iChange))
```

Comment:
Can you explain what's up with the absolute values here? Shouldn't a negative change be possible?